hazelcast/src/test/java/com/hazelcast/core/PipeliningTest.java

```diff
@@ -53,6 +56,45 @@ public void add_whenNull() throws InterruptedException {
         pipelining.add(null);
     }
 
+
+    @Test
+    public void testInterrupt() throws Exception {
+        Pipelining<String> pipelining = new Pipelining<String>(1);
+        pipelining.add(mock(ICompletableFuture.class));
+
+        TestThread t = new TestThread() {
+            @Override
+            public void doRun() throws Throwable {
+                pipelining.add(mock(ICompletableFuture.class));
```

Comment:
You need to mark the pipelining as `final` as the code needs to be JDK6 compatible.

Reply:
Done